### PR TITLE
[itowns] Add new definition

### PR DESCRIPTION
**Pull request by Desplandis**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## itowns ([<kbd>unpkg</kbd>](https://unpkg.com/browse/itowns@latest/))
was missing the following properties:
1. C3DTILES_LAYER_EVENTS
1. ColorLayersOrdering
1. GlobeLayer
1. LabelLayer
1. OrientedImageLayer

<details>
<summary>as well as these 3 other properties...</summary>
<p>PlanarLayer, TiledGeometryLayer, legacyGLTFLoader</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 38136ff319c0d80c8eac35fadfa7c1ea25b50171
</p>

File: types/itowns/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// Type definitions for itowns 2.39
+// Project: https://itowns.github.io/
+// Definitions by: Bouillaguet Quentin <https://github.com/desplandis>
+//                 Madec Germerie <https://github.com/mgermerie>
```
Comment (peterblazejewicz) [inline types/itowns/index.d.ts:4]: you'll have to get this person explicit ACK here, to be listed as an owner

Comment (Desplandis) [reply types/itowns/index.d.ts:4]: @peterblazejewicz Is this sufficient to ACK in this PR thread or shall he commit itself to the list of definitions owners?

Comment (peterblazejewicz) [reply types/itowns/index.d.ts:4]: it's enough to read ack here on the thread somwhere, thanks! Iv'e seen emoji reaction, but that's not the same as ack imo

Comment (mgermerie) [reply types/itowns/index.d.ts:4]: Hi @peterblazejewicz ! Thanks for your review. I hereby give my ack to be listed as an owner of the package.

**Review by peterblazejewicz — APPROVED** [review]
@Desplandis thanks!
LGTM!

**Review by mgermerie — APPROVED** [review]

**Comment by Desplandis** [comment]
Ready to merge

**Pull request merged**